cajita/unit_test/CMakeLists.txt

```diff
@@ -76,11 +76,14 @@ endmacro()
 set(SERIAL_TESTS
   GlobalMesh
   IndexSpace
-  Parallel
   ParameterPack
   Splines
   )
 
+if(NOT Kokkos_ENABLE_OPENMPTARGET) #FIXME_OPENMPTARGET
+  list(APPEND SERIAL_TESTS Parallel)
+endif()
```

Comment:
I would prefer conditionally removing from the list

Reply:
Done